File: sin/sin.js
// ATOMIC FUNCTION:abs
// TAGS: math，numeric
// LANG: node.js
// FUNCTION: Math.abs(val, precision)

var start_val=Math.PI/2;
var expected_end_val=1;

var end_val=Math.sin(start_val);

// TEST
if(end_val==expected_end_val){
    console.log('PASS');
}else{
    console.log('FAIL');
}

File: sin/sin.py

# ATOMIC_FUNCTION:  sin(val)
# TAGS: math，numeric
# LANG:  python
# FUNCTION: sin(val)
import math
start_val = math.pi/2
expected_end_val = 1

end_val = math.sin(start_val)


# TEST
if(end_val == expected_end_val):
    print("PASS")
else:
    print("FAIL")





